upcloud/load_balancer.go

```diff
@@ -323,9 +327,10 @@ type LoadBalancerActionHTTPReturn struct {
 	Payload     string `json:"payload,omitempty"`
 }
 
-// LoadBalancerActionHTTPRedirect represents 'http_redirect' action
+// LoadBalancerActionHTTPRedirect represents 'http_redirect' action. Only either Location or Scheme should be defined.
```

Comment:
I was thinking if the SDK itself should enforce only one of "Location" or "Scheme", but perhaps that would be over-engineering. Especially, if other similar cases are currently validated only by the API and not by the SDK.

Reply:
Yeah, I think it is better to keep this part simple and rely on errors given by API.

The validation could be done in TF provider and `upctl`. If those end up very similar we could then move the validation here.